homeassistant/components/qvrpro/camera.py

```diff
@@ -0,0 +1,65 @@
+"""
+Support for viewing camera streams from QVR Pro.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/camera.qvrpro
```

Comment:
Please remove this as it is unnecessary and does not [describe what the file is about](https://developers.home-assistant.io/docs/en/development_guidelines.html#file-headers).